brain/compartmentReportView.h

```diff
@@ -78,6 +78,22 @@ class CompartmentReportView
      */
     BRAIN_API std::future<brion::Frames> load(double start, double end);
 
+    /** Load frames between start and end time stamps.
+     *
+     * @param start the start time stamp
+     * @param end the end time stamp
+     * @param stride the time stride
```

Comment:
I trust your judgement wrt to names, so name it here whatever you like (accidentaly, BluePy exposes this parameter as 'step'; in my request I named it 'stride' for the reason I stated above: the previous C++ layer was accepting frame numbers and not times).